backend/internal/service/utils/pagination.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+type Pagination struct {
+	Page  int `query:"page"`
+	Limit int `query:"limit"`
+}
+
+func (p *Pagination) Validate() map[string]string {
```

Comment:
could we set defaults for page and limit (maybe 1 and 10 respectively)? i feel like we shouldn't error if the user doesn't specify anything

Reply:
Sure, I can make it default to page 1 limit 10. Suppose user includes bad numbers tho, like page -2 limit -3, should this error or just default to page 1 limit 10

Reply:
hmm kinda subjective but i prefer erroring in that case so its clear to the user that they can't do this (rather than just magically cleaning up their mistake)